shotover-proxy/Cargo.toml

```diff
@@ -11,6 +11,7 @@ license = "Apache-2.0"
 [features]
 # Include WIP alpha transforms in the public API
 alpha-transforms = []
+cpp-driver-tests = []
```

Comment:
The `cpp` is a bit confusing, maybe one of:
* `cassandra-cpp-driver-tests` be specific about which driver we are enabling
* `external-dependency-integration-tests` be as generic as possible

Reply:
Went with `cassandra-cpp-driver-tests` until we need to be generic.